bip-0174.mediawiki

```diff
@@ -483,6 +483,50 @@ The currently defined per-input types are defined as follows:
 | 0, 2
 | [[bip-0371.mediawiki|371]]
 |-
+| MuSig2 Participant Public Keys
+| <tt>PSBT_IN_MUSIG2_participant_PUBKEYS = 0x19</tt>
```

Comment:
`_participant_` should be in capital letters

Reply:
Fixed